app/over_react_redux/todo_client/lib/src/utils.dart

```diff
@@ -0,0 +1,43 @@
+import 'package:react/react_client/js_backed_map.dart';
+
+extension DeepConvert on JsBackedMap {
+  bool _isJsMap(dynamic value) => value is JsMap && value is! Function;
+
+  /// Traverses the provided [jsMap] and recursively
+  /// converts any nested [JsMap]s to [JsBackedMap]s.
+  ///
+  /// Inverse of [deepJsObject].
+  JsBackedMap deepConvertFromJs(JsMap jsMap) {
```

Comment:
This shouldn't be an instance method since it has nothing to do with the instance being operated on. I think a top-level function would work better.